src/main/common/filter.c

```diff
@@ -309,3 +309,25 @@ void simpleLPFilterInit(simpleLowpassFilter_t *filter, int32_t beta, int32_t fpS
     filter->beta = beta;
     filter->fpShift = fpShift;
 }
+
+void meanAccumulatorAdd(meanAccumulator_t *filter, const int8_t newVal)
+{
+    filter->accumulator += newVal;
+    filter->count++;
+}
+
+int8_t meanAccumulatorCalc(meanAccumulator_t *filter, const int8_t defaultValue)
+{
+    if (filter->count) {
+        int8_t retVal = filter->accumulator / filter->count;
+        meanAccumulatorInit(filter);
+        return retVal;
+    }
+    return defaultValue;
+}
+
+void meanAccumulatorInit(meanAccumulator_t *filter)
+{
+    filter->accumulator = 0;
+    filter->count = 0;
+}
```

Comment:
I'm not sure `laggedMovingAverage` is the best fit for this data. What the RX needs to return is the average SNR since the last time it reported. This can be up to 128 entries, and laggedMovingAverage would need to store 128 floats of sliding window which is a lot of storage just to get the average. The RX never needs to sample it in the middle of a period, and must not carry over any values from the previous period into the new period. This would happen for any LQ <100. The number of entries also changes dynamically during a connection, meaning the filter will have to be reset and possibly make its value unusable due to a low number of entries.

laggedMovingAverage would require a tremendous amount of RAM and also not do a good job of what it needs to do, so I think this it is better to have a different mechanism.

Reply:
Carried the code over from ELRS codebase to have consistent behavior. The SNR value is sent back to the transmitter and used for dynamic power calculation. With a different way of filtering it might give inconsistent results compared to standalone RXs.

Reply:
Per the comments above, it's better to be consistent with the standalone eLRS RX when measuring telemetry values. Saving RAM is a bonus.

Reply:
Yeah okay, fair enough.